packages/docs/src/page-configs/getting-started/configuration-guide/code-examples/colors-custom-classes.ts

```diff
@@ -0,0 +1,21 @@
+export const colorsCustomClasses = `
+// main.js
+import { VuesticPlugin } from 'vuestic-ui'
+import 'vuestic-ui/dist/vuestic-ui.css'
+
+const app = createApp(App)
+
+app.use(VuesticPlugin, {
+  colors: { ... },
+  colorsCustomClasses: [
```

Comment:
Don't we should keep colorsCustomClasses inside of colors object?

Reply:
It depends on it, but `colorsClasses` is separate service.